extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -404,6 +425,123 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
+   * Returns a string-valued config data property set by the publisher in the
+   * tag in a `data-` attribute or in the `json` attribute. Throws an error if
+   * the value is not a string.
+   *
+   * @param {string} name the name of the config data property
+   * @return {?string} the value of the config data property, or `null` if it's
+   *     absent
+   */
+  getStringData(name) {
+    if (name in this.jsonConfig_) {
```

Comment:
We should make it clear that we prioritize json over explicit attributes... this is actually very interesting as it relates to RTC (FYI @bradfrizzell).  Can we user().warn if the value is specified in both json and attribute but they differ?